docs/references/expo/custom-auth-flows/expo-impersonation.mdx

```diff
@@ -0,0 +1,491 @@
+---
+description: Learn how to build custom Impersonation into your Expo application
+---
+
+# Impersonation in Expo
+
+<Callout type="warning">
+The following flows are built assuming your Clerk app is using the default instance settings. If you have changed your instance settings, you may need to make changes to the examples.
+</Callout>
+
+User Impersonation is a powerful feature to support your customers with. Currently, from the Clerk Dashboard, [you can impersonate your users](https://clerk.com/docs/custom-flows/user-impersonation#impersonate-a-user-from-the-clerk-dashboard) and see through their eyes inside of your web applications. In this example, we’ll learn how to build impersonation into your mobile application through [Clerk’s custom flows](https://clerk.com/docs/custom-flows/overview). 
+
+The example we’ll build will still need some work before it goes into your app. For example, we will build impersonation directly into the dashboard, whereas you should build your impersonation into an admin only dashboard, where all of your users are listed & can be interacted with. You will want to restrict this so **only employees of your company or those who should have access to other user's information** can use this implementation.
+
+## Before you start
+
+Before continuing, please go through the [Expo Quickstart](https://clerk.com/docs/quickstarts/expo) and utilize the repo as a starting point for this example.
+
+```bash
+git clone https://github.com/clerk/clerk-expo-quickstart
+```
+
+Since we will be using `expo-router`'s experimental API routes, you will need to follow the [Expo docs](https://docs.expo.dev/router/reference/api-routes/#deployment) to deploy your routes to a domain. This is so we have access to our `CLERK_SECRET_KEY` that we need to pass to our impersonation endpoint. 
+
+For this project, you need to configure a few settings from the Clerk dashboard. 
+
+1. Configure your Clerk instance to enable Email and Password 
+    1. Navigate to the Clerk Dashboard and go to **User & Authentication > [Email, Phone, and Username](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username)** in the sidebar menu.
+    2. Ensure that **Email address** is required, and **Username** is not required
+    3. Ensure that **Password** is required
```

Comment:
```suggestion
1. You must configure your Clerk instance to enable **Email and Password** authentication:
    1. Navigate to the Clerk Dashboard and go to **User & Authentication > [Email, Phone, and Username](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username)** in the sidebar menu.
    1. Ensure that **Email address** is required, and **Username** is not required
    1. Ensure that **Password** is required
1. Install the following dependencies:
    ```bash
    npm i expo-linking && npm i -D @clerk/types
    ```
```
- Adding this item to the numbered list defined with the edits above, and changing the numbers to `1`
- Moving the dependency installation instructions into the "before you start" section